Description: Hardcode version
 Ugly hack: rollup is required for importing package.json. This removes circular
 build dependency. This should be fixed properly when we have latest version of
 rollup in the archive and a proper staged build is used.
Forwarded: not-needed
Reviewed-By: Xavier Guimard <yadd@debian.org>
Last-Update: 2021-01-29

--- a/src/rollup/rollup.ts
+++ b/src/rollup/rollup.ts
@@ -1,4 +1,4 @@
-import { version as rollupVersion } from 'package.json';
+const rollupVersion = '2.42.1';
 import Bundle from '../Bundle';
 import Graph from '../Graph';
 import { ensureArray } from '../utils/ensureArray';
--- a/src/utils/PluginContext.ts
+++ b/src/utils/PluginContext.ts
@@ -1,4 +1,4 @@
-import { version as rollupVersion } from 'package.json';
+const rollupVersion = '2.42.1';
 import Graph from '../Graph';
 import {
 	NormalizedInputOptions,
